profiles/arch/powerpc/package.use.mask

```diff
@@ -1,6 +1,11 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Florian Schmaus <flo@geekplace.eu> (2020-09-30)
+# Rubber's testsuite depends on dev-tex/pythontex, which is not
+# keyworded on ppc.
+dev-tex/rubber test
```

Comment:
We should probably do this in profiles/arch/powerpc/ppc32 instead (the structure is confusing and will be fixed soon). Looks ok, just remember to file a bug afterwards for keywording.